docs/getting-started-guides/juju.md

```diff
@@ -36,27 +36,32 @@ Juju dependencies for your platform.
 
 ### With Docker
 
+If you are not using Ubuntu or prefer the isolation of Docker, you may
+run the following:
+
 > While this is a common target, the charmbox flavors of images are
 > unofficial, and should be treated as Experimental. If you encounter any issues
 > turning up the Kubernetes cluster with charmbox, please file a bug on the
 > respective issue tracker [here](https://github.com/juju-solutions/charmbox/issues)
 
-If you are not using Ubuntu or prefer the isolation of Docker, you may
-run the following:
-
 ```shell
 mkdir ~/.juju2
 sudo docker run -v ~/.juju2:/home/ubuntu/.local/share/juju -ti jujusolutions/charmbox:devel
 ```
 
-At this point from either path you will have access to the `juju
-bootstrap` command. However we will need to configure the credentials for the
- Juju cloud provider before we can proceed.
+### Configure Juju to point a cloud
+
+At this point you have access to the Juju client. If you wish to use a cloud,
```

Comment:
I really like this refactoring, but "if you wish to use a cloud" seems wrong in this context. They're going to *have* to use a cloud unless they're on that very specific 1.25 local:kvm provider. Perhaps we should change just that sentence to "Before you can leverage juju client, you just first configure the credentials for the juju provider"